core/src/main/scala/org/ensime/indexer/ClassfileIndexer.scala

```diff
@@ -50,9 +50,15 @@ trait ClassfileIndexer {
       superName: String, interfaces: Array[String]
     ): Unit = {
 
+      val signatureClass = try {
```

Comment:
I'd like to treat all failures as a bug instead of silently ignoring. In 2.0 this is fine.